### DISCUSSION TITLE
Amiga CD32 on Koriki???

### DISCUSSION THREAD

**Discussion by Tekkno303** [rg35xx-cfw/Koriki#62] [General]
How to run games from Amiga CD32 / CDTV on Koriki rg35xx ??
I copied the folder with bios and kickstarts from GarlicOS to Koriki. I copied the games from Amiga 500 to the Amiga 500 (koriki OS) folder. I copied games from Amiga CD32 (different versions: iso+bin, m3u and folders, lha) to a folder on the second partition with KorikiOS.
Amiga 500 games are displayed in Koriki in the "Home Computers" => Amiga tab. Games from Amiga CD32 copied to a folder on the SD card do not appear in the Koriki menu at all. They don't exist :/ How to run it?

**Comment by acmeplus** [comment]
The configuration for amigacd32 is not included. You have two options, one is to add amigacd32 folder to the amiga section, or to create a new one.

To add them to the existing amiga collection, edit the file ``system/.simplemenu/section_groups/home computers.ini`` and change the amiga section to add the folder where you have the amigacd32 to the romDirs:
```ini
[AMIGA]
execs = /userdata/system/.simplemenu/launchers/puae2021_libretro,/userdata/system/.simplemenu/launchers/uae4arm_libretro,/userdata/system/.simplemenu/launchers/puae_libretro
romDirs = /userdata/roms/amiga500/,/userdata/roms/amiga/,/userdata/roms/amigacd32/
romExts = .adf,.adz,.dms,.fdi,.ipf,.hdf,.hdz,.lha,.slave,.info,.cue,.ccd,.nrg,.mds,.iso,.chd,.uae,.m3u,.zip,.7z,.rp9
```

To create a new category in the consoles section, edit the file ``system/.simplemenu/section_groups/consoles.ini`` and modify the first line to add AMIGACD32 there:
```ini
consoleList = AMIGACD32,NINTENDO 64,DREAMCAST,ODYSSEY2,NES,FDS,SNES,MSU-1,SGB,VIRTUAL BOY,SEGA SG-1000,MASTER SYSTEM,SEGA GENESIS,SEGA CD,SEGA 32X,ATARI 2600,ATARI 5200,ATARI 7800,INTELLIVISION,NEO GEO CD,PC ENGINE,PC ENGINE
```

then add the AMIGACD32 section:
```ini
[AMIGACD32]
execs = /userdata/system/.simplemenu/launchers/puae2021_libretro,/userdata/system/.simplemenu/launchers/uae4arm_libretro,/userdata/system/.simplemenu/launchers/puae_libretro
romDirs = /userdata/roms/amigacd32/
romExts = .zip,.m3u
```

Note that if you use the second approach you will need to know how to modify the theme to add the new background, etc.

**Reply by paladinperry** [reply]
thank you for this, I wanted to add some of these also
